diff -urN  openmw-master/CI/before_script.msvc.sh.orig openmw-master/CI/before_script.msvc.sh
--- openmw-master/CI/before_script.msvc.sh.orig
+++ openmw-master/CI/before_script.msvc.sh
@@ -917,7 +917,8 @@ printf "Qt ${QT_VER}... "
 		mkdir Qt
 		cd Qt
 
-		run_cmd "${DEPS}/aqt-venv/${VENV_BIN_DIR}/aqt" install ${QT_VER} windows desktop "win${BITS}_msvc${QT_MSVC_YEAR}${SUFFIX}"
+ 		run_cmd "${DEPS}/aqt-venv/${VENV_BIN_DIR}/aqt" install ${QT_VER} windows desktop "win${BITS}_msvc${QT_MSVC_YEAR}${SUFFIX} -b http://ftp1.nluug.nl/languages/qt"
+
 
 		printf "  Cleaning up extraneous data... "
 		rm -rf Qt/{aqtinstall.log,Tools}
